linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/rule.yml

```diff
@@ -107,8 +114,26 @@ ocil: |-
     {{% endif %}}
 
 fixtext: |-
+    {{% if 'ubuntu' in product %}}
```

Comment:
also mention the nmcli fix

Reply:
As decided, we are moving away from the nmcli fix.